File: Modul 5. Fungsi/PRAK502/PRAK502-2310817120003-NatalieGraceKatiandagho.py
def hitung(nilai1, nilai2):
    hitung_total = nilai1 - nilai2
    if hitung_total < 0:
        return hitung_total * -1
    else:
        return hitung_total  

def mutlak(angka):
    if angka < 0:
        return -angka
    return angka

a, c, b, d = map(int, input().split())

hasil = hitung(a, b) + hitung(c, d)
print(mutlak(hasil))

File: Modul 5. Fungsi/PRAK502/PRAK502-2310817120003-NatalieGraceKatiandagho.c
#include <stdio.h>
#include <math.h>

int hitung(int nilai1, int nilai2)
{
    int hitung_total = nilai1 - nilai2;
    if (hitung_total < 0)
    {
        return hitung_total * -1;
    }
}

int mutlak(int angka)
{
    if (angka < 0)
    {
        return -angka;
    }
    return angka;
}

int main()
{
    int a, b, c, d;
    scanf("%d", &a);
    scanf("%d", &c);
    scanf("%d", &b);
    scanf("%d", &d);
    int Hasil = hitung(a, b) + hitung(c, d);
    printf("%d", mutlak(Hasil));
    return 0;
}
